.github/workflows/vsphere-e2e.yaml

```diff
@@ -33,6 +33,8 @@ jobs:
           buildConfig: "offline"
         - os: "rocky 9.1"
           buildConfig: "offline"
+        - os: "rocky 9.1"
+          buildConfig: "basic"
```

Comment:
Curious why you added this?

Reply:
I wanted to test the "VMs" network to test image building for networked usecase. So I added it. This will be removed anyways with rocky-9.5 in next PR.